compiler/rustc_codegen_llvm/src/back/write.rs

```diff
@@ -423,6 +423,18 @@ fn get_pgo_sample_use_path(config: &ModuleConfig) -> Option<CString> {
         .map(|path_buf| CString::new(path_buf.to_string_lossy().as_bytes()).unwrap())
 }
 
+fn get_instr_profile_output_path(
+    config: &ModuleConfig,
+    out_directory: &PathBuf,
+) -> Option<CString> {
+    if config.instrument_coverage {
+        let output_path = out_directory.join("default_%m_%p.profraw");
```

Comment:
I don't think we want to join with `out_directory` here because (at least for Cargo) that will cause the profraw file to be placed in the `target\{profile}\deps` folder and not the current working directory like you would expect. Just specifying the filename without a directory component should be sufficient.

Reply:
Understood, will change. Thanks for the review!